app/components/chart-footnotes.tsx

```diff
@@ -67,17 +67,30 @@ export const ChartFootnotes = ({
                 color="primary"
                 variant="text"
                 size="small"
+                startIcon={
+                  <Icon
+                    name={
+                      isChartTablePreview
+                        ? getChartIcon(chartConfig.chartType)
+                        : "table"
+                    }
+                    size={16}
+                  />
+                }
                 onClick={() => setIsChartTablePreview(!isChartTablePreview)}
-                sx={{ width: "20px", minWidth: 0, padding: 0 }}
+                sx={{ padding: 0, fontWeight: "regular" }}
```

Comment:
Nit: Am wondering why we need to put a fontWeight here, maybe we would need to have a new button variant ? Or have the text variant of the button as fontWeight: regular ?

Reply:
It's because there is a `fontWeight: bold` when `size="small"` - I can create another variant or change this? I am not sure is size small should implicate a bold font weight, IMO it's slightly confusing, what do you think? 😅

Reply:
Ok, so I'll change that and use sx={{fontWeight: "bold"}} for the stepper buttons.